src/sagemaker/serve/utils/tuning.py

```diff
@@ -1,5 +1,7 @@
 """Holds mixin logic to support deployment of Model ID"""
```

Comment:
From a glance it looks like a lot of dupe code in this file, do we need it all or can we reuse?

Reply:
+1. We seem to have _pretty_print_results and _pretty_print_results_tgi https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/serve/utils/tuning.py#L21 , can't we reuse generic one?